test/utils/shippable/integration.sh

```diff
@@ -67,8 +67,12 @@ if [ "${controller_shared_dir}" ]; then
     cp -a "${SHIPPABLE_BUILD_DIR}" "${controller_shared_dir}"
 fi
 
+# Enabling colored output in Shippable console
+color_settings="--env ANSIBLE_FORCE_COLOR=1"
```

Comment:
Forcing color output causes causes the ```blocks``` integration test to fail. You can test this with:

```IMAGE=ansible/ansible:ubuntu1604 TARGET=blocks test/utils/shippable/integration.sh```

This is due to the test using grep on the playbook output. It probably wouldn't hurt to modify the test to work with or without color output.

I do like having the option to force color output, as it makes it possible to have color when running tests locally and saving them to a log file. I think the ideal solution would be to have force color the default, but being able to disable it when running integration.sh locally. You'll have to fix the tests to work with color though.